README-GIANO-INJECTION.md

```diff
@@ -131,6 +140,11 @@ Manage public key storage and retrieval.
 **getPublicKeyByCredentialId**: Retrieve stored public key coordinates
 **onCredentialKey**: Store public key coordinates for a credential
 
+#### `submitUserOperation()` (Optional)
+Override for custom user operation submission.
+
+This function is only included in the injection when `enableBackendSubmission` is set to `true` in the options. When present, it receives complete signed user operations and should handle their submission to your backend or bundler service, returning the transaction receipt.
+
```

Comment:
This also needs to be updated, we need to return the user operation hash, so the receipt can be awaited separately